package.json

```diff
@@ -35,7 +35,7 @@
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r protos build/",
     "compile-protos": "compileProtos src",
-    "docs": "jsdoc -c .jsdoc.js",
+    "docs": "NODE_OPTIONS=--max-old-space-size=8192 jsdoc -c .jsdoc.js",
```

Comment:
@sofisl fun trick, you can pass options to the node bin through `NODE_OPTIONS`, `jsdoc` is actually a shorthand for:

```
node ./node_modules/.bin/jsdoc
```

So you were passing arguments to `jsdoc` rather than to `node`.